src/test/run-pass/consts/const-construct-call.rs

```diff
@@ -0,0 +1,108 @@
+// Test that constructors are considered to be const fns
```

Comment:
Move this to `src/test/ui/consts/min_const_fn/` since you are affecting stable.

Reply:
Since we are feature gating per https://github.com/rust-lang/rust/pull/61209#discussion_r287591005, instead move things to a dedicated folder in `ui/consts` and then let's move it later to `min_const_fn`.